src/bundle/Resources/config/fieldtype_services.yaml

```diff
@@ -145,3 +145,10 @@ services:
             - '%ibexa.field_type.richtext.converter.edit.xhtml5.resources%'
             - '@ibexa.config.resolver'
 
+    Ibexa\Bundle\FieldTypeRichText\Command\MigrateNamespacesCommand:
+        autowire: true
```

Comment:
Any good reason for avoiding autowire and autoconfigure?

Reply:
Technically, Symfony recommends not using `autowire` and `autoconfigure` due to the need to compute during container compilation (which does not affect prod, and dev when there no container changes). We currently do not follow this recommendation, because it does indeed remove a lot of our service definition declarations/code.

Just wanted to clarify.